bugbug/commit_features.py

```diff
@@ -128,6 +128,54 @@ def __call__(self, commit, **kwargs):
         }
 
 
+class source_code_file_metrics(object):
+    name = "metrics on source code file"
+
+    def __call__(self, commit, **kwargs):
+        return {
+            "Average cyclomatic": commit["average_cyclomatic"],
+            "Average number of unique operands": commit[
+                "average_halstead_unique_operands"
+            ],
+            "Average number of operands": commit["average_halstead_operands"],
+            "Average number of unique operators": commit[
+                "average_halstead_unique_operators"
+            ],
+            "Average number of operators": commit["average_halstead_operators"],
+            "Average number of source loc": commit["average_source_loc"],
+            "Average number of logical loc": commit["average_logical_loc"],
+            "Maximum cyclomatic": commit["maximum_cyclomatic"],
+            "Maximum number of unique operands": commit[
+                "maximum_halstead_unique_operands"
+            ],
+            "Maximum number of operands": commit["maximum_halstead_operands"],
+            "Maximum number of unique operators": commit[
+                "maximum_halstead_unique_operators"
+            ],
+            "Maximum number of operators": commit["maximum_halstead_operators"],
+            "Maximum number of source loc": commit["maximum_source_loc"],
+            "Maximum number of logical loc": commit["maximum_logical_loc"],
+            "Minimum cyclomatic": commit["minimum_cyclomatic"],
+            "Minimum number of unique operands": commit[
+                "minimum_halstead_unique_operands"
+            ],
+            "Minimum number of operands": commit["minimum_halstead_operands"],
+            "Minimum number of unique operators": commit[
+                "minimum_halstead_unique_operators"
+            ],
+            "Minimum number of operators": commit["minimum_halstead_operators"],
+            "Minimum number of source loc": commit["minimum_source_loc"],
+            "Minimum number of logical loc": commit["minimum_logical_loc"],
+            "Sum of number of operands": commit["sum_halstead_operands"],
```

Comment:
Use `Total` instead of `Sum of`, as there is some code which simplifies the names and relies on `Maximum`, `Minimum`, `Total` and `Average` being there